css/index.css

```diff
@@ -1,139 +1,287 @@
+/* Mobile style */
+
 body {
-  background-color: #EDE0D4;
+  margin: 0;
```

Comment:
The requirements don't mention it, but you could always wrap this in a media query as well.